Purpose: Fix to work with python3
Upstream: Shebang is still python

--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # Frr Reloader
 # Copyright (C) 2014 Cumulus Networks, Inc.
 #
@@ -802,7 +802,7 @@
                 add_exit_vrf = False
 
         if ctx_keys[0].startswith('vrf') and line:
-            if line is not "exit-vrf":
+            if line != "exit-vrf":
                 add_exit_vrf = True
                 prior_ctx_key = (ctx_keys[0])
             else:
